ipinfo/cmd_grepip.go

```diff
@@ -13,6 +13,12 @@ var completionsGrepIP = &complete.Command{
 	Flags: map[string]complete.Predictor{
 		"-o":                 predict.Nothing,
 		"--only-matching":    predict.Nothing,
+		"-c":                 predict.Nothing,
+		"--include-cidrs":    predict.Nothing,
+		"-r":                 predict.Nothing,
```

Comment:
Let's not use a short flag for this - we tried to copy some common flag naming conventions from normal `grep` (hence `-o`), and `-r` is too easily associated with `recursive lookup`.